youtube_dl/extractor/tubitv.py

```diff
@@ -97,6 +90,10 @@ def _real_extract(self, url):
                 'url': self._proto_relative_url(sub_url),
             })
 
+        is_live = None
+        if video_data.get('tags'):
+            is_live = 'live' in video_data.get('tags')
+
```

Comment:
```suggestion
        is_live = ('live' in video_data.get('tags') or []) or None

```
or, a more complex version that won't crash if `video_data['tags']` isn't iterable or `None`,
```suggestion
        is_live = traverse_obj(video_data, 'tags', expected_type=lambda x: 'live' in x) or None

```